input/pagecontent/reference.md

```diff
@@ -1373,7 +1379,7 @@ To create a mapping, the keywords `Mapping`, `Source`, `Target` and `Id` are use
   Source:   USCorePatient
   Target:   "http://unknown.org/Argonaut-DQ-DSTU2"
   Id:       argonaut-dq-dstu2
-  * -> Patient
+  * -> "Patient"
   * extension[USCoreRaceExtension] -> "Patient.extension[http://fhir.org/guides/argonaut/StructureDefinition/argo-race]"
   * extension[USCoreEthnicityExtension] -> "Patient.extension[http://fhir.org/guides/argonaut/StructureDefinition/argo-ethnicity]"
   * extension[USCoreBirthSexExtension] -> "Patient.extension[http://fhir.org/guides/argonaut/StructureDefinition/argo-birthsex]"
```

Comment:
It may be helpful in this example to show one or two examples of the `{comment}` and `{mime-type}` syntax in context.